src/applications/personalization/profile360/selectors.js

```diff
@@ -48,6 +49,40 @@ export function selectVet360FailedTransactions(state) {
   return state.vet360.transactions.filter(isFailedTransaction);
 }
 
+export function selectVet360PendingCategoryTransactions(state, type) {
+  const {
+    vet360: {
+      transactions,
+      fieldTransactionMap
+    }
+  } = state;
+
+  const existsWithinFieldTransactionMap = (transaction) => {
+    const transactionId = transaction.data.attributes.transactionId;
+
+    return Object
+      .keys(fieldTransactionMap)
+      .some(fieldName => {
+        const transactionRequest = fieldTransactionMap[fieldName];
+        return transactionRequest.transactionId === transactionId;
+      });
+  };
+
```

Comment:
A helper for determining whether a transaction contains a mapping to an exact field.